website/models.py

```diff
@@ -7,4 +7,24 @@ class User(models.Model):
     last_name = models.CharField(max_length=64)
     email_address = models.CharField(max_length=254)
     password = models.CharField(max_length=64)
-    salt = models.CharField(max_length=64)
\ No newline at end of file
+    salt = models.CharField(max_length=64)
+
+
+class Notification(models.Model):
+    id = models.CharField(max_length=64)
+    name = models.CharField(max_length=64)
+    notification_group = models.CharField(max_length=64)
```

Comment:
Guess I'm confused.  Are these models for the database or the UI?  I was assuming the UI but it sounds like you are saying this is for the database.